src/main/webapp/app/iris/settings/iris-settings-update/iris-settings-update.component.ts

```diff
@@ -76,7 +76,8 @@ export class IrisSettingsUpdateComponent implements OnInit, DoCheck, ComponentCa
     loadIrisSettings(): void {
         this.isLoading = true;
         this.loadIrisSettingsObservable().subscribe((settings) => {
-            this.loadIrisModels();
+            //this.loadIrisModels();
```

Comment:
I assume this might be related to the commented-out code in the tests.

Reply:
Yes. Will be reenable, once PyrisV2 supports this.